region/lowernorfair/west/Golden Torizo's Room.json

```diff
@@ -479,6 +622,35 @@
         "FIXME: As an alternative to doing a Crystal Flash, Golden Torizo could be used as a farm, with canRiskPermanentLossOfAccess."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Morph, IBJ",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphLongIBJ",
+        {"or": [
+          "f_DefeatedGoldenTorizo",
+          {"and": [
+            "canTrickyDodgeEnemies",
+            "h_canArtificialMorphJumpIntoIBJ"
```

Comment:
OR: can staggered IBJ.
Double IBJ is fast enough to outpace GT, or you can get him to jump away and double IBJ up.
Note: When double IBJing you have to switch to single IBJS to overload PLMS before touching the crumbles, or do it twice.